connector_create.go

```diff
@@ -145,6 +172,7 @@ func (s *ConnectorCreateService) Do(ctx context.Context) (ConnectorCreateRespons
 	headers := make(map[string]string)
 	headers["Authorization"] = s.c.authorization
 	headers["Content-Type"] = "application/json"
+	headers["Accept"] = "application/json;version=2"
```

Comment:
Maybe we should define a constant for `application/json;version=2`?

Reply:
@shirolimit - sorry I just saw your comments now. 👍🏻 I'll add it in the next release, ok? Thank you.

Reply:
No worries, I was too late with my review :)